data/sgid-index/index.html

```diff
@@ -3279,7 +3279,8 @@ <h6 id='show_filters' class='pointer'><i class="fas fa-search"></i> Show Filters
                 <td data-th="category" class="category">transportation</td>
                 <td data-th="name" class="name"><a href="https://data-uplan.opendata.arcgis.com/search?q=aadt">AADT</a></td>
                 <td data-th="agency" class="agency">UDOT</td>
-                <td data-th="description" class="description">Average Annual Daily Traffic (AADT)  on road sections of the State

+                <td data-th="description" class="description">Average Annual Daily Traffic (AADT)  on road sections of the State
+

```

Comment:
Do you know what is causing all of these whitespace changes in this file? Do you have some sort of formatter enabled or something? It would be good if you could remove these changes. It just makes the commit messy. I can help if you need it.

Reply:
Hmm...that's weird, I don't think I even touched that line.  It just changed the line related to the open source places.  Should we just revert that file to normal and let PR #1975 (which I thing was auto-created) make that change?

Reply:
That's a good idea. I don't know much about the sgid-index stuff but it seems like this PR shouldn't be touching that file. Let's keep it to #1975.